google-cloud-clients/google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettingsTest.java

```diff
@@ -188,20 +192,23 @@ public void readRowsIsNotLostTest() {
     builder.readRowSettings().setRetryableCodes(Code.ABORTED, Code.DEADLINE_EXCEEDED);
 
     assertThat(builder.readRowsSettings().getIdleTimeout()).isEqualTo(Duration.ofMinutes(5));
-    assertThat(builder.readRowsSettings().getRetryableCodes())
-        .containsAllOf(Code.ABORTED, Code.DEADLINE_EXCEEDED);
+    assertThat(builder.readRowsSettings().getRetryableCodes().containsAll(retryCodeList));
```

Comment:
The assertion is wrong here & everywhere else in this change is incorrect.
instead of :
`assertThat(builder.readRowsSettings().getRetryableCodes().containsAll(retryCodeList));`
either use this:
`assertThat(builder.readRowsSettings().getRetryableCodes().containsAll(retryCodeList)).isTrue();`
Or a better approach would be:
`assertThat(builder.readRowsSettings().getRetryableCodes()).containsAtLeast(retryCodeList);`

Please make sure to actual object is inside assertThat method

Reply:
done